include/torch-mlir/Dialect/Torch/Transforms/Passes.h

```diff
@@ -32,6 +32,11 @@ struct TorchLoweringPipelineOptions
   // If this option is false, only do the bare minimum for correctness.
   Option<bool> optimize{*this, "optimize", llvm::cl::desc("Do optimizations."),
                         llvm::cl::init(true)};
+  
+  // If this option is false, decompose complex operations.
+  // If this option is true, skip decomposition of complex operations.
+  Option<bool> nodecompose{*this, "nodecompose", llvm::cl::desc("No decomposition."),
```

Comment:
nit: can you call this "decompose-complex-ops" with a default of "true"?

Reply:
sure, done